lib/from_openstudio/material/window_gas_mixture.rb

```diff
@@ -0,0 +1,74 @@
+# *******************************************************************************
+# Honeybee OpenStudio Gem, Copyright (c) 2020, Alliance for Sustainable
+# Energy, LLC, Ladybug Tools LLC and other contributors. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# (1) Redistributions of source code must retain the above copyright notice,
+# this list of conditions and the following disclaimer.
+#
+# (2) Redistributions in binary form must reproduce the above copyright notice,
+# this list of conditions and the following disclaimer in the documentation
+# and/or other materials provided with the distribution.
+#
+# (3) Neither the name of the copyright holder nor the names of any contributors
+# may be used to endorse or promote products derived from this software without
+# specific prior written permission from the respective party.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDER(S) AND ANY CONTRIBUTORS
+# "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+# THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER(S), ANY CONTRIBUTORS, THE
+# UNITED STATES GOVERNMENT, OR THE UNITED STATES DEPARTMENT OF ENERGY, NOR ANY OF
+# THEIR EMPLOYEES, BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT
+# OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+# STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+# OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+# *******************************************************************************
+
+require 'honeybee/material/window_gas_mixture'
+require 'to_openstudio/model_object'
+
+module Honeybee
+  class EnergyWindowMaterialGasMixture
+
+    def self.from_material(material)
+        # create an empty hash
+        hash = {}
+        hash[:type] = 'EnergyWindowMaterialGasMixture'
+        # set hash values from OpenStudio Object
+        hash[:identifier] = material.nameString
+        hash[:thickness] = material.thickness
+        hash[:gas_types] = []
+        hash[:gas_fractions] = []
+        hash[:gas_types] << material.gas1Type
+        hash[:gas_fractions] << material.gas1Fraction
+        number_of_gases = material.numberofGasesinMixture
+        # Check and add if more than 1 gas is present
+        if number_of_gases > 1
```

Comment:
The honeybee `EnergyWindowMaterialGasMixture` always has to have between 2 and 4 gasses. You can see this in the schema docs:
https://www.ladybug.tools/honeybee-schema/model.html#tag/energywindowmaterialgasmixture_model

So you should only be using this translation pathway if there are 2 or more gasses.

Also all of these `if` statements could be written a lot more cleanly with a single for loop like so:
```
(1..number_of_gases).each do |n|
    hash[:gas_types] << material.send('gas' + n.to_s + 'Type')
end
```
(I think that's the correct equivalent of [getting an attribute by its text name in Ruby](https://stackoverflow.com/questions/786412/what-is-the-ruby-equivalent-of-pythons-getattr))